### DISCUSSION TITLE
Corner only/dashed tile highlight.

### DISCUSSION THREAD

**Discussion by geheur** [runelite/runelite#16857] [Ideas]
![image](https://github.com/runelite/runelite/assets/41499327/8214bcee-27c5-4e91-9ae1-fd5fe24bd7f4)

I like using this for true tile, maybe it would be good in the core client as an option for tile indicators, npc indicators and ground markers?

https://github.com/geheur/runelite/tree/dashed-and-corner-indicators messy but in case anyone wants to try it out.

Some tile markers:
![image](https://github.com/runelite/runelite/assets/41499327/b40d4f5d-8cd6-4a97-b483-812ee00f3349)

**Comment by Nightfirecat** [comment]
I don't see why not

**Comment by geheur** [comment]
I made a PR for this for tile indicators https://github.com/runelite/runelite/pull/16962

Would be pretty easy to do the same for ground markers.

Npc indicators might be tricker because I don't know if I like dashed tile indicators; I haven't seen it enough on 2x2 or larger npcs to know if I want corners or dashed for those. I would also really like the dashes to be tile-aligned. Also, better npc indicators added corners and dashed tile markers so it's already available if people want it.
